ci/build.sh

```diff
@@ -69,6 +79,46 @@ if [ "$TRAVIS_BRANCH" != "master" ] && [ "$TRAVIS_EVENT_TYPE" == "push" ] && [ -
   else
       echo "No changes in 'fluentd-sumologic.yaml.tmpl'."
   fi
+
+  # Generate override yaml file for chart dependencies if changes are made to values.yaml file
+  if git diff --name-only $TRAVIS_COMMIT_RANGE | grep -q -i "values.yaml"; then
+    echo "Detected changes in 'values.yaml', generating file fluent-bit-overrides.yaml..."
+    fluent_bit_start=`grep -n "fluent-bit:" deploy/helm/sumologic/values.yaml | head -n 1 | cut -d: -f1`
+    fluent_bit_start=$(($fluent_bit_start + 2))
+    fluent_bit_end=`grep -n "## Configure prometheus-operator" deploy/helm/sumologic/values.yaml | head -n 1 | cut -d: -f1`
+    fluent_bit_end=$(($fluent_bit_end - 1))
+    
+    echo "Copy 'values.yaml' from line $fluent_bit_start to line $fluent_bit_end to 'fluent-bit-overrides.yaml'"
+    echo "# This file is auto-generated." > deploy/helm/fluent-bit-overrides.yaml
+    # Copy lines of fluent-bit section and remove indention from values.yaml
+    sed -n "$fluent_bit_start,${fluent_bit_end}p" deploy/helm/sumologic/values.yaml | sed 's/  //' >> deploy/helm/fluent-bit-overrides.yaml
+    # Remove release name from service name
+    sed -i 's/collection-sumologic/fluentd/' deploy/helm/fluent-bit-overrides.yaml
+
+    echo "Detected changes in 'values.yaml', generating file prometheus-overrides.yaml..."
+    prometheus_start=`grep -n "prometheus-operator:" deploy/helm/sumologic/values.yaml | head -n 1 | cut -d: -f1`
+    prometheus_start=$(($prometheus_start + 2))
+   
+    echo "Copy 'values.yaml' from line $prometheus_start to end to 'prometheus-overrides.yaml'"
+    echo "# This file is auto-generated." > deploy/helm/prometheus-overrides.yaml
+    # Copy lines of fluent-bit section and remove indention from values.yaml
```

Comment:
`fluent-bit` should be `prometheus` in this comment

Reply:
nice catch!